content/en/docs/developer_guide/project/configuration/project_configuration.md

```diff
@@ -11,6 +11,36 @@ date: 2021-10-01
 
 {{% alert title="Note" color="primary" %}}API key values and session objects are auto-generated when you create a new project.{{% /alert %}}
 
+### Configuration files
+
+{{% variables/apibuilder_prod_name %}} will look for two types of configuration files in the **`./conf`** directory of the application, either "default" or "local", for example:
+
+* default.js
+* local.js
+* oracle.default.js
+* oracle.local.js
+
+Files ending in `default.js` are part of the application, can be added to source control, and will be released into production, so these must never contain sensitive information, such as passwords.
+
+Files ending in `local.js` are never be added to source control, and will not be released into production, so these files can contain sensitive information such as passwords.
```

Comment:
**suggestion** Maybe we should be clear to users this isn't magic and is under the user's control in the application. Specifically:

These files are ignored by git and docker using `.gitignore`. and `.dockerignore` files that come with your project. If you use alternative source control or deployment methods you should configure them to ignore these files.

I wonder if we have this in the security guide.